Sources/PCCurrentUser.swift

```diff
@@ -1285,27 +1285,24 @@ public typealias PCRoomsCompletionHandler = ([PCRoom]?, Error?) -> Void
 #if os(iOS) || os(macOS)
 import PushNotifications
 
-private let pushNotifications: PushNotifications = PushNotifications.shared
-
 extension PCCurrentUser {
     /**
      Start PushNotifications service.
      */
     public func enablePushNotifications() {
-        pushNotifications.start(instanceId: self.v6Instance.id)
-        self.setUser(self.id)
-        ChatManager.registerForRemoteNotifications()
-    }
+        let beamsClient = PushNotifications.init(instanceId: self.v6Instance.id)
```

Comment:
Is this calling the `PushNotifications` constructor, or is there a method called `init`?

Reply:
.init is just explicitly calling the initialiser, we have removed this word (and it still does the same thing)